test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -2902,11 +2902,15 @@ class test_ucp_sockaddr_protocols_err_sender
         send_buf.pattern_fill(3, size);
 
         /* Complete receive operations */
+        ucs::ptr_vector<mem_buffer> recv_bufs;
```

Comment:
maybe just delete in `rtag_complete_check_data_cbx`?

Reply:
do you mean delete buffer in `rtag_complete_check_data_cbx` instead of defining `ptr_vector`?
if yes, I think it is not so good method, because other tests use `rtag_complete_check_data_cbx` and they don't expect that `mem_buffer` will be deleted there

Reply:
yes, I see only 2 places where the callback is used.

Reply:
so, it means that we will need to have two different callbacks.
I'd keep using `ptr_vector` to release buffer on a caller side who owns this buffer

Reply:
or allocate in 2nd place with operator `new` too

Reply:
@yosefe wdyt?